store/src/java-test/com/zimbra/cs/datasource/DataSourceManagerTest.java

```diff
@@ -0,0 +1,119 @@
+package com.zimbra.cs.datasource;
```

Comment:
The copyright section is missing?

Reply:
@ykomiyam thanks for spotting this. Turns out a few of soap tests are missing license block.